imagetest/fixtures.go

```diff
@@ -79,6 +80,25 @@ func (t *TestWorkflow) LockProject() {
 	t.lockProject = true
 }
 
+// WaitForVMQuota appends a list of quotas to the wait for vm quota step. Quotas with a blank region will be populated with the region corresponding to the workflow zone.
+func (t *TestWorkflow) WaitForVMQuota(qa ...*daisy.QuotaAvailable) error {
```

Comment:
Right now we are passing in a single qa, do you see us needing more than one?

Reply:
It's a variadic argument, this function can handle multiple quotas.

Reply:
I know, my question is do we have such a use case?

Reply:
I misread your comment, I'm sorry. I don't think we have any use right now, everything that needs quotas needs multiple quotas but they're all stored in such a way that we call for them one by one.

Reply:
Yep, just that the method signature should reflects its use case (ideally).

Reply:
Makes sense. Done.